### DISCUSSION TITLE
Blocked Account

### DISCUSSION THREAD

**Discussion by adedola12** [vercel/community#3719] [Account Issue]
### Question

Hello please i tried creating an account using this git hub account but it keeps saying my account has been blocked. please help me out, i have sent mail also but no response.

**Comment by amyegan** [comment]
Due to the sensitive nature of account information, we cannot safely discuss it in this public forum. For more info, please check this guide: [Why has my account or deployment been paused?](https://vercel.com/guides/why-is-my-account-deployment-blocked)
